scss/mixins/_debugcolors.scss

```diff
@@ -0,0 +1,35 @@
+.grid_15 {
+  background-color: lightcoral;
+}
+
+.grid_12 {
+  background-color: lightslategrey;
+}
+
+.grid_9 {
+  background-color: lightgray;
+}
+
+.grid_6 {
+  background-color: lightpink;
+}
+
+.grid_5 {
+  background-color: darkcyan;
+}
+
+.grid_4 {
+  background-color: darkgray;
+}
+
+.grid_3 {
+  background-color: lightyellow;
+}
+
+.grid_2 {
+  background-color: lightgreen;
+}
+
+.grid_1 {
+  background-color: lightblue;
+}
```

Comment:
As an exercise:

I'm pretty sure we can create this using a loop and a map. Could you give this a go?

Reply:
```
$map: (
  1: orange,
  2: red,
  5: blue
);

@each $grid_kolom, $kleur in $map {
  grid_#{$grid_kolom} {
    color: $kleur;
  }
}
```